### PR TITLE
Use options.host instead of options.hostname (#416)

### LINKED ISSUE #416: Opencensus/instrumentation-http http.js hostname undefined
Please answer these questions before submitting a bug report.

### What version of OpenCensus are you using?
0.0.9

### What version of Node are you using?
v.10.13.0

### What did you do?
used instrumentation-http plugin for tracing

### What did you see instead?
```shell
(node:56170) UnhandledPromiseRejectionWarning: Error: Unsupported type : undefined
    at createAttributeValue (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/@opencensus/exporter-stackdriver/build/src/stackdriver-cloudtrace-utils.js:134:19)
    at createAttributesBuilder (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/@opencensus/exporter-stackdriver/build/src/stackdriver-cloudtrace-utils.js:113:29)
    at Object.createAttributes (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/@opencensus/exporter-stackdriver/build/src/stackdriver-cloudtrace-utils.js:49:31)
    at StackdriverTraceExporter.createSpan (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/@opencensus/exporter-stackdriver/build/src/stackdriver-cloudtrace.js:95:56)
    at rootSpans.forEach.rootSpan (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/@opencensus/exporter-stackdriver/build/src/stackdriver-cloudtrace.js:78:36)
    at Array.forEach (<anonymous>)
    at StackdriverTraceExporter.<anonymous> (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/@opencensus/exporter-stackdriver/build/src/stackdriver-cloudtrace.js:76:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/@opencensus/exporter-stackdriver/build/src/stackdriver-cloudtrace.js:19:58)
    at propagateAslWrapper (/Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/async-listener/index.js:504:23)
    at /Users/jinsuk/Code/w4/chatbots/service/backoffice/node_modules/async-listener/index.js:541:70
(node:56170) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
```

### PR DESCRIPTION
**Pull request by jinmel**
Fixes #416

### CODE REVIEW

**Comment by vmarchaud** [comment]
I'm not sure you saw the comment i made on the commit, but i believe this patch also belong in the `https` instrumentation since it's the same underlying server. Also i believe i patched it too in my implementations

File: packages/opencensus-instrumentation-http/src/http.ts [hunk 1]
```diff
@@ -377,8 +377,8 @@ export class HttpPlugin extends BasePlugin {
           const tags = new TagMap();
           tags.set(stats.HTTP_CLIENT_METHOD, {value: method});
 
-          if (options.hostname) {
```
Comment (mayurkale22) [inline packages/opencensus-instrumentation-http/src/http.ts]: I think ```hostname``` is preferred over ```host```. I would suggest to add below line.
```const host = options.hostname || options.host || 'localhost';``` WDYT?

Comment (mayurkale22) [reply packages/opencensus-instrumentation-http/src/http.ts]:
```
const host = options.hostname || options.host || 'localhost';
span.addAttribute(HttpPlugin.ATTRIBUTE_HTTP_HOST, host);
```
This change will also fix failing build.

Comment (jinmel) [reply packages/opencensus-instrumentation-http/src/http.ts]: thanks!

**Pull request merged**